userguide/intro.rst

```diff
@@ -218,7 +218,8 @@ These screen options have changed:
   :menuselection:`Tasks --> Cloud Sync Tasks --> ADD`.
 
 * The :guilabel:`Name` field has been added to
-  :menuselection:`Tasks --> Replication Tasks --> Add`.
+  :menuselection:`Tasks --> Replication Tasks --> Add`. Replicaiton
+  names are unique.
```

Comment:
github will not let me enter a suggestion here.
"are" does not explain that the system no longer allows duplicates.  Use "must be unique."